<?xml version="1.0"?>
<launch>
    <arg name="roomba" default="roomba1" />

	<node pkg="multi_localizer" type="bbox_msg_recorder" name="bbox_msg_recorder" >
		<remap from="bbox_in" to="$(arg roomba)/bounding_boxes" />

		<param name="FILE_PATH" value="$(find multi_localizer)/record/$(arg roomba)" />	
	</node>

</launch>